packages/web-components/src/components/link-list/link-list.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { html, customElement, property, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './link-list.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Link list.
+ *
+ * @element dds-link-list
+ */
+@customElement(`${ddsPrefix}-link-list`)
+class DDSLinkList extends LitElement {
+  @property({ type: String })
+  type = 'default';
```

Comment:
Could you define `default` here as an enum, like [one for footer size](https://github.com/carbon-design-system/ibm-dotcom-library/blob/93db39f/packages/web-components/src/components/footer/footer.ts#L41-L45)?